src/arduino-nano-element.ts

```diff
@@ -0,0 +1,277 @@
+          <g id="pins-pin1" fill="#333" filter="url(#solderPlate)">
+            <rect x="${15.5 * 2.54 - 0.875}" y="${0.5 * 2.54 - 0.875}" width="1.75" height="1.75" />
+            <rect x="${15.5 * 2.54 - 0.875}" y="${6.5 * 2.54 - 0.875}" width="1.75" height="1.75" />
+          </g>
+          <rect x="2.54" width="${15 * 2.54}" height="2.54" />
+          <rect x="2.54" y="${6 * 2.54}" width="${15 * 2.54}" height="2.54" />
+        </g>
+
+        <!-- programming header -->
+        <g id="pgm-header" fill="url(#pins-pth-1.00)" stroke="#eee" stroke-width=".1">
+          <g id="pgm-pin1" stroke="none" fill="#333" filter="url(#solderPlate)">
+            <rect x="${16.5 * 2.54 - 0.875}" y="${4.5 * 2.54 - 0.875}" width="1.75" height="1.75" />
+          </g>
+          <rect x="${15 * 2.54}" y="${2 * 2.54}" width="${2 * 2.54}" height="${3 * 2.54}" />
+        </g>
+
+        <!-- USB mini type B -->
+        <g id="usb-mini-b" stroke-width=".1" paint-order="stroke fill">
+          <g fill="#333" filter="url(#solderPlate)">
+            <rect x=".3" y="3.8" width="1.6" height="9.8" />
+            <rect x="5.5" y="3.8" width="1.6" height="9.8" />
+            <rect x="7.3" y="7.07" width="1.1" height=".48" />
+            <rect x="7.3" y="7.82" width="1.1" height=".48" />
+            <rect x="7.3" y="8.58" width="1.1" height=".48" />
+            <rect x="7.3" y="9.36" width="1.1" height=".48" />
+            <rect x="7.3" y="10.16" width="1.1" height=".48" />
+          </g>
+          <rect x="-1.4" y="4.8" width="8.9" height="7.8" fill="#999" stroke-width=".26" />
+          <rect x="-1.25" y="5" width="8.6" height="7.4" fill="#ccc" stroke="#bbb" />
+          <g fill="none" stroke="#333" stroke-width=".26" stroke-linecap="round">
+            <path d="m2.6 5.9h-3.3v0.9h3.3m0 3.7h-3.3v0.9h3.3M-0.6 7.6l4.3 0.3v1.5l-4.3 0.3" />
+            <path d="m3.3 7.9v1.5" stroke-width="1" stroke-linecap="butt" />
+            <path d="m6 6.4v4.5" stroke-width=".35" />
+          </g>
+        </g>
+
+        <!-- LEDs -->
+        <g transform="translate(27.7 5)">
+          <use xlink:href="#led-body" />
+          ${ledTX &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#ff8080" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 7)">
+          <use xlink:href="#led-body" />
+          ${ledRX &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#80ff80" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 9)">
+          <use xlink:href="#led-body" />
+          ${ledPower &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#80ff80" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 11)">
+          <use xlink:href="#led-body" />
+          ${led13 &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#ffff80" filter="url(#ledFilter)" />`}
+        </g>
+
+        <!-- reset button -->
+        <g
```

Comment:
how about adding a `tabindex="0"` attribute, so that the reset will be focusable by keyboard (using the tab key?)

Reply:
I've taken a shot at this in the latest commit.  Much googling was required.  I'm still not 100% happy.  I would like the button to lose focus on mouseleave, but I'm struggling to make that happen.

I've left the LED in pending your feedback re the LED/gradient.

Reply:
Shared my feedback. What was the rationale behind losing focus on mouseleave?

Reply:
Thanks.  Losing focus on mouseleave is purely for appearance.  It didn't feel right that the button looked different before/after clicking it, until it lost focus somehow.